### PR TITLE
Fix the test failure

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
Fix the test failures of #51 

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: jira7/jira_data_mappings.bal [hunk 1]
```diff
@@ -233,27 +233,6 @@ function issueRequestToJson(IssueRequest source) returns json {
...
-    sourceMap["assigneeName"] = "";
-    
-    foreach var f in sourceMap.keys() {
-        var result = json.convert(sourceMap[f]);
-        if (result.length == 0) {
-            sourceMap[f] = "";
-        } else {
-            target.fields[f] = result;
-        }
-    }
```
Comment (Manuri) [inline jira7/jira_data_mappings.bal]: Why are we removing this code?

File: jira7/jira_data_mappings.bal [hunk 2]
```diff
@@ -280,3 +259,164 @@ function jsonToIssueComment(json jcomment) returns IssueComment {
...
+    ? sourceGroupStruct.projectCategory.self.toString() : "";
+    targetProject.projectCategory.id = sourceGroupStruct.projectCategory.id != null
+    ? sourceGroupStruct.projectCategory.id.toString() : "";
+    targetProject.projectCategory.name = sourceGroupStruct.self != null ? sourceGroupStruct.self.toString() : "";
+    targetProject.projectCategory.description = sourceGroupStruct.projectCategory.description != null
+    ? sourceGroupStruct.projectCategory.description.toString() : "";
+    targetProject.issueTypes = convertToIssueTypes(<json[]>sourceGroupStruct.issueTypes);
+    targetProject.components = convertToComponents(<json[]>sourceGroupStruct.components);
+    targetProject.versions = convertToVersions(<json[]>sourceGroupStruct.versions);
+    return targetProject;
```
Comment (Manuri) [inline jira7/jira_data_mappings.bal]: Can't we directly convert the json to Project? Check other places as well.
https://ballerina.io/learn/by-example/json-struct-map-conversion.html

File: jira7/jira_data_mappings.bal (hunk 1 above)
Comment (kalaiyarasiganeshalingam) [reply jira7/jira_data_mappings.bal]: The IssueRequest record has defined four attribute in the https://github.com/wso2-ballerina/module-jira/blob/master/jira7/jira_types.bal#L326. Here we added the four attribute into the target. Hence no need to do the further configuration. So I have removed.

File: jira7/jira_data_mappings.bal (hunk 2 above)
Comment (kalaiyarasiganeshalingam) [reply jira7/jira_data_mappings.bal]: Yes,We can't do that. When I converted the json data to Project, I got the following error

incompatible stamp operation: 'json' value cannot be stamped as 'wso2/jira7:0.8.16:Project'

**Pull request merged**